source/operations/install-deploy-manage/deploy-minio-multi-node-multi-drive.rst

```diff
@@ -104,6 +104,14 @@ Storage Requirements
    :start-after: start-storage-requirements-desc
    :end-before: end-storage-requirements-desc
 
+Memory Requirements
+~~~~~~~~~~~~~~~~~~~
+
+Starting with :minio-release:`RELEASE.2024-01-28T22-35-53Z`, MinIO pre-allocates 2GiB of system memory at startup.
```

Comment:
```suggestion
Starting with :minio-release:`RELEASE.2024-01-28T22-35-53Z`, MinIO pre-allocates 2GiB of system memory per node at startup.
```

Reply:
This is leaving a full URL for the release in the generated page. Is that expected?
Or should it be a hyperlink with just the version number as text?